src/App.js

```diff
@@ -1,24 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import { LoginPage } from './components/Login';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
boycha why are you delete this section

Reply:
this is because we don't need this code I have change with a new component called loginpage.